Expense Tracker/script.py

```diff
@@ -0,0 +1,100 @@
+from tkinter import ttk
+from tkinter import messagebox
+from tkinter import *
+from tkinter.ttk import Notebook
+from tkcalendar import DateEntry
+import sqlite3
+
+
+def Addexpense():
+    x = Edate.get()
+    y = Title.get()
+    z = Eexpense.get()
+    data = [x,y,z]
+    TVExpense.insert('','end',values=data)
+    with db:
+        c = db.cursor()
+        c.execute("INSERT INTO expense(Dates, Items, Expense) VALUES(?,?,?)",(x,y,z))
+       
+
+
+def show():
+    connt=sqlite3.connect('expense.db')
+    cursor=connt.cursor()
+    cursor.execute("SELECT * FROM expense")
+    for row in cursor.fetchall():
+        print(row)
+
+def delete():
+    with db:
+        c = db.cursor()
+        c.execute("DROP table expense")
+        db.commit()
+
+db = sqlite3.connect('expense.db')
+c = db.cursor()
+
+# uncomment this when running the script for the first time or when deleting it
```

Comment:
This is not the way of implementing the code "comment-uncomment". Refer to this implementation for databases: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/899

Also, delete the database uploaded: expense.db